nimble/host/include/host/ble_gap.h

```diff
@@ -982,6 +983,54 @@ struct ble_gap_event {
         } periodic_transfer;
 #endif
 
+#if MYNEWT_VAL(BLE_BIGINFO_REPORTS)
+        /**
+         * Represents a periodic advertising sync transfer received. Valid for
+         * the following event types:
+         *     o BLE_GAP_EVENT_BIGINFO_REPORT
+         */
+        struct {
+            /** Synchronization handle */
+            uint16_t sync_handle;
+
+            /** Number of present BISes */
+            uint8_t bis_cnt;
+
+            /** Number of SubEvents */
+            uint8_t nse;
+
+            /** ISO Interval */
+            uint16_t iso_interval;
+
+            /** Burst Number */
+            uint8_t bn;
+
+            /** Pre-Transmission Offset */
+            uint8_t pto;
+
+            /** Immediate Repetition Count */
+            uint8_t irc;
+
+            /** Maximum PDU size */
+            uint16_t max_pdu;
+
+            /** Service Data Unit Interval */
+            uint8_t sdu_interval[3];
```

Comment:
`uint32_t` will be easier to handle by app
also move below `max_sdu` so we don't waste space on alignment unnecessarily